src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -220,6 +229,18 @@ export type PathologyReportPDF = {
     url: string;
 };
 
+export type confidenceDataMapType = {
```

Comment:
Types should be named in title case, not camel case. Also, we probably don't need the `Type` suffix in the name. 

```suggestion
export type ConfidenceDataMap = {
```

Maybe not export this type for now, since it seems like we only use it in this store. If we really want to import this type in other places, maybe better to export this in a utility class or something.

Reply:
Removed the export